File: arrays-1.py
# https://leetcode.com/problems/find-the-duplicate-number/
count = {}
for num in nums:
  if num not in nums:
    count[num] = 1
  else:
    count[num] += 1
 for key,value in count.items():
  if value>1:
    return key
return -1

File: arrays-1.c
// https://leetcode.com/problems/find-the-duplicate-number/submissions/
int findDuplicate(int* nums, int numsSize){
    int* count = calloc(sizeof(int),numsSize);
    for(int i=0;i<numsSize;i++){
        count[nums[i]]++;
        if(count[nums[i]]>1)
            return nums[i];
    }
    return -1;
}
